shared/vehicles.lua

```diff
@@ -349,8 +349,8 @@ QBShared.Vehicles = {
         ['brand'] = 'Overflod',
         ['model'] = 'zeno',
         ['price'] = 1350000,
-        ['category'] = 'coupes',
-        ['categoryLabel'] = 'Coupes',
+        ['category'] = 'super',
+        ['categoryLabel'] = 'Super',
```

Comment:
A blista isnt a super car?

Reply:
![image](https://user-images.githubusercontent.com/20475516/207592238-fb946db9-8660-4caa-a1d5-fa2fa31c1a2c.png)
thats not for the blista